neural_network/activation_functions/binary_step.py

```diff
@@ -0,0 +1,36 @@
+"""
+This script demonstrates the implementation of the Binary Step function.
+
+It's an activation function in which the neuron is activated if the input is positive,
```

Comment:
```suggestion
It's an activation function in which the neuron is activated if the input is positive or 0,
```
"Positive" means "> 0"